docs/sql-data-sources-jdbc.md

```diff
@@ -43,34 +43,39 @@ following command:
 
 Spark supports the following case-insensitive options for JDBC. The Data source options of JDBC can be set via:
 * the `.option`/`.options` methods of
-  *  `DataFrameReader`
-  *  `DataFrameWriter`
+  * `DataFrameReader`
+  * `DataFrameWriter`
 * `OPTIONS` clause at [CREATE TABLE USING DATA_SOURCE](sql-ref-syntax-ddl-create-table-datasource.html)
 
 For connection properties, users can specify the JDBC connection properties in the data source options.
 <code>user</code> and <code>password</code> are normally provided as connection properties for
 logging into the data sources.
 
 <table class="table">
-  <tr><th>Property Name</th><th>Meaning</th></tr>
+  <tr><th><b>Property Name</b></th><th><b>Default</b></th><th><b>Meaning</b></th><th><b>Scope</b></th></tr>
```

Comment:
Do we need `<b>` inside `<th>`?

Reply:
Actually I just followed the rule in existing files, and seems like It's a little bit more thicker 😅

- Before <b>

<img width="336" alt="Screen Shot 2021-06-04 at 2 24 59 PM" src="https://user-images.githubusercontent.com/44108233/120750365-35238b00-c541-11eb-8faa-90e94c6a0a9d.png">

- After <b>

![Screen Shot 2021-06-04 at 2 25 34 PM](https://user-images.githubusercontent.com/44108233/120750375-394fa880-c541-11eb-8e66-b3b28a34cb6f.png)

Reply:
Thank you for checking! Looks better. 😄